### PR TITLE
Github integration scan messages enrichment

### PR DESCRIPTION
**Pull request by cborla**
|Related issue|Documentation|Testing Manual|
|---|---|---|
|https://github.com/wazuh/wazuh/issues/13784 https://github.com/wazuh/wazuh/issues/13943| [Documentation](https://github.com/wazuh/wazuh-documentation/pull/5357)| https://github.com/wazuh/wazuh-qa/issues/3013|

## Description
The goal of this change is encrease the information about how Github integration works. 
2 new messages are added, one when first scan runs, with the legend `Bookmark updated to '2022-06-16T18:21:58Z', waiting '60' seconds to run first scan`, where the first date is the UTC date as the API request to Github server, and the `'60' seconds` are the interval time. Second message is similar than first, it only change the word `next` instead the word `first`, and it shows when bookmark is updated. `Bookmark updated to '2022-06-16T18:21:58Z', waiting '60' seconds to run next scan`

## Configuration options
ossec.conf 

```
<github>
    <enabled>yes</enabled>
    <interval>1m</interval>
    <time_delay>30s</time_delay>
    <curl_max_size>1M</curl_max_size>
    <only_future_events>yes</only_future_events>
    <api_auth>
        <org_name>dummy</org_name>
        <api_token>dummy</api_token>
    </api_auth>
    <api_parameters>
        <event_type>all</event_type>
    </api_parameters>
</github>

```
## Logs

```
2022/06/16 15:22:28 wazuh-modulesd:github[303215] wm_github.c:225 at wm_github_execute_scan(): DEBUG: Scanning organization: 'binarybeast2022'
2022/06/16 15:22:28 wazuh-modulesd:github[303215] wm_github.c:257 at wm_github_execute_scan(): DEBUG: Bookmark updated to '2022-06-16T18:21:58Z', waiting '60' seconds to run first scan
2022/06/16 15:23:28 wazuh-modulesd:github[303215] wm_github.c:257 at wm_github_execute_scan(): DEBUG: Bookmark updated to '2022-06-16T18:22:58Z', waiting '60' seconds to run next scan

```

- Compilation without warnings in every supported platform
  - [x] Linux
  - [x] Windows
  - [x] MAC OS X
- [x] Source installation
- [x] Package installation
- [x] Source upgrade
- [x] Package upgrade
- [x] Review logs syntax and correct language

- Memory tests for Linux
  - [ ] Coverity
  - [x] Valgrind (memcheck and descriptor leaks check)

### CODE REVIEW

File: src/wazuh_modules/wm_github.c [hunk 1]
```diff
@@ -349,6 +355,15 @@ STATIC void wm_github_execute_scan(wm_github *github_config, int initial_scan) {
                     wm_github_scan_failure_action(&github_config->fails, current->org_name, event_types[event_types_it], error_msg, github_config->queue_fd);
                 } else {
                     org_state_struc.last_log_time = new_scan_time;
+
+                    if (isDebug()) {
+                        memset(new_scan_time_str, '\0', 80);
```
Comment (TomasTurina) [inline src/wazuh_modules/wm_github.c:360]: This code is not necessary, just the debug log.

**Review by TomasTurina — CHANGES_REQUESTED** [review]

File: src/wazuh_modules/wm_github.c (hunk 1 above)
Comment (cborla) [reply src/wazuh_modules/wm_github.c:360]: I agree

File: src/wazuh_modules/wm_github.c [hunk 2]
```diff
@@ -349,6 +355,7 @@ STATIC void wm_github_execute_scan(wm_github *github_config, int initial_scan) {
                     wm_github_scan_failure_action(&github_config->fails, current->org_name, event_types[event_types_it], error_msg, github_config->queue_fd);
                 } else {
                     org_state_struc.last_log_time = new_scan_time;
+                    mtdebug1(WM_GITHUB_LOGTAG, "Bookmark updated to '%s', waiting '%ld' seconds to run next scan", new_scan_time_str, github_config->interval);
```
Comment (TomasTurina) [inline src/wazuh_modules/wm_github.c:358]: This line should be in an else after the following if.

**Review by TomasTurina — CHANGES_REQUESTED** [review]

**Review by TomasTurina — APPROVED** [review]

**Comment by jmv74211** [comment]
## QA review

- **Type**: Manual testing
- **Status**: Approved  🟢 
- **Testing issue**: https://github.com/wazuh/wazuh-qa/issues/3013
- **Comments**: Everything seems to work properly. It has been proposed a suggestion [here](https://github.com/wazuh/wazuh/issues/13944#issuecomment-1164168755)

**Pull request merged**